packages/showcase/src/user-controls/common-snippets/common-snippets.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, Input, ViewEncapsulation, inject } from "@angular/core";
+import { CommonModule } from "@angular/common"; // Import CommonModule
+import { Router, RouterLink } from "@angular/router";
+import { RouterCommonLinksService } from "../../services/router-links.service";
+
+@Component({
+  selector: "common-snippets",
+  template: `
```

Comment:
Since this template will grow and expand, it is best to move it to the .html file.